frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/ReferenceTab/ReferenceTab.test.tsx

```diff
@@ -79,5 +88,6 @@ const renderReferenceTab = ({
         ...componentProps,
       }}
     />,
+    { queries: { ...queriesMock, getOptionListIds: mockOptionListIds } },
```

Comment:
the queries passed from `renderWithProviders` will spread the queriesMock anyway so it should be enough to do this:

```suggestion
    { queries: getOptionListIds },
```

(If you change the name of the mock function)